src/components/InterestForm/index.tsx

```diff
@@ -86,46 +125,51 @@ export default function InterestForm({ caseData }: { caseData: CaseListing }) {
     <FormContainer>
       <H3>Submit Interest</H3>
       {submitted ? (
-        <P>We have received your submission!</P>
+        <P>Your submission has been received!</P>
       ) : (
-        <>
-          <FormQuestion $color={COLORS.greyDark}>
-            What role(s) are you applying for?
-          </FormQuestion>
-          <RadioGroup>
-            {radioOptions.map(option => (
-              <Radio key={option}>
-                <RadioInput
-                  id={`radio${option}`}
-                  type="radio"
-                  name="radioOptions"
-                  value={option}
-                  checked={rolesInterested === option}
-                  onChange={event => setRolesInterested(event.target.value)}
-                />
-                <RadioLabel htmlFor={`radio${option}`}>{option}</RadioLabel>
-              </Radio>
-            ))}
-            {missingInfo && rolesInterested === '' && (
-              <ErrorText>Must select your preferred role</ErrorText>
+        <Flex $gap="30px" $direction="column">
+          {(listingData.listing_type === 'CASE' ||
+            listingData.listing_type === 'INT') && (
+            <DateInput
+              label="What is the earliest date you can contact the client?"
+              required
+              error={getErrorText()}
+              name="startDate"
+              value={startDate}
+              setValue={setStartDate}
+            />
+          )}
+          {listingData.listing_type === 'CASE' && !interpretation && (
+            <RadioGroup
+              name="reason"
+              required
+              value={needsInterpreter}
+              setValue={setNeedsInterpreter}
+              options={['Yes', 'No']}
+              label="Do you need language interpretation help for the client?"
+              error={
+                missingInfo
+                  ? 'Must select whether you need language support'
```

Comment:
Can we rephrase this error message so it uses the same wording as the question? i.e. "language interpretation help" or whatever we end up using in the question instead of "language support"

Reply:
let's update this to "Must select whether you need language interpretation assistance" to match Pragya's suggested wording for the question.

Reply:
sg!